templates/taxi/manufacturer_list.html

```diff
@@ -23,11 +25,22 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url "taxi:manufacturer-update" manufacturer.pk %}">UPDATE</a>
+              </td>
+              <td>
+                <a href="{% url "taxi:manufacturer-delete" manufacturer.pk %}">DELETE</a>
+              </td>
           </tr>
         {% endfor %}
       </table>
 
     {% else %}
         <p>There are no manufacturers in the service.</p>
     {% endif %}
+  <button type="button" class="btn btn-primary"
```

Comment:
When you use buttons you are adding Javascript to the mix which is not preferred  way to work with django URL routing system in templates. Using hyperlinks is easier to understand and maintain and will work for users with JavaScript turned off
```
<a class="btn btn-primary" style="margin: 10px; padding: 10px 20px;" href="{% url 'taxi:manufacturer-create' %}">
    CREATE
</a>